src/test/java/com/ays/admin_user/service/impl/AdminUserRegisterServiceImplTest.java

```diff
@@ -307,13 +250,11 @@ void givenExistingPhoneNumberFromAdminUserRegisterRequest_whenAdminUserExist_the
         // Then
         Assertions.assertThrows(
                 AysAdminUserAlreadyExistsByPhoneNumberException.class,
-                () -> adminUserRegisterService.register(mockAdminUserRegisterRequest)
+                () -> adminUserRegisterService.completeRegistration(applicationId, mockAdminUserRegisterApplicationCompleteRequest)
         );
 
         Mockito.verify(adminUserRegisterApplicationRepository, Mockito.times(1))
```

Comment:
Aşağıdaki gibi suggestion kısmını da ekleyebilir miyiz?
```suggestion
        // Verify
        Mockito.verify(adminUserRegisterApplicationRepository, Mockito.times(1))
```

Reply:
Ekledim.